### ISSUE TITLE
Site rows with the same name should be combined when possible

### ISSUE THREAD

**Issue by njarboe** [earthref/MagIC#305]
It would be better to have combined site rows when it makes sense. For example in https://www2.earthref.org/MagIC/16338 each site has two entries. They can be combined as most of the columns in each row that has a value is empty in the other row. Where there are values in both rows they are the same, except method codes, which could be concatenated without loosing any info, I believe. Maybe we need a data migration cleanup project to fix this and the other problems we come across like the normal polarity default.

**Comment by rminnett** [comment]
In most of these cases it appears as though the method codes and the criteria are the reasons the rows aren't merged during an upgrade. We decided these shouldn't be merged automatically since the method codes, for example, might refer to only the data entered in one of the rows and not the other.

**Comment by njarboe** [comment]
Ok. I'm going through some older contributions for a paper Cromwell et al. is working on and if I find many contributions that should have the site rows combined I can write a tool to do that. Unless your 2.5 to 3.0 converter has some code or the ability to do that. Take a sites table that has 2 rows with the same site name and then merge them as described above?

**Comment by rminnett** [comment]
Yes, https://github.com/earthref/MagIC/blob/9da58a54e82ca53e977d2528e7d11b3f3e128eee/lib/modules/magic/upgrade_contribution.js#L1155 and https://github.com/earthref/MagIC/blob/9da58a54e82ca53e977d2528e7d11b3f3e128eee/lib/modules/magic/upgrade_contribution.js#L1209 do that during the upgrade, but determining row pairwise orthogonality was done very conservatively to avoid mismatching data and metadata (we agreed on merging descriptions, though, for example).
